### PR TITLE
Add WSL install file

### PR DESCRIPTION
**Pull request by dgud**

### CODE REVIEW

**Comment by ferd** [comment]
Seeing the doc disappear, does this means that windows usage without WSL is becoming unsupported?

**Comment by dgud** [comment]
Not yet, but we don't currently test the old ones, we have not used cygwin for ages so 
we don't even know if that still works.

**Comment by dgud** [comment]
Re-added the old desc with a disclaimer a the top.

File: HOWTO/INSTALL-WIN32.md [hunk 1]
```diff
@@ -4,40 +4,31 @@ How to Build Erlang/OTP on Windows
...
-involved are still named win32. Some occurances of the name win64 are 
-however present. The installation file for a 64-bit Windows version of 
-Erlang, for example, is `otp_win64_%OTP-REL%.exe`.
+This section describes how to build the Erlang emulator and the OTP
+libraries on Windows. Note that the Windows binary releases are still
+a preferred alternative if one does not have Microsoft’s development
+tools and/or don’t want to install WSL.
+
+The instructions apply to Windows 10 (v.1809 and later) supporting the
+WSL.1 (Windows Subsystem for Linux v.1) and using Ubuntu 18.4 release.
```
Comment (peterdmv) [inline HOWTO/INSTALL-WIN32.md:13]: 18.04

File: HOWTO/INSTALL-WIN32.md [hunk 2]
```diff
@@ -649,65 +181,52 @@ tools:
         export $ERL_TOP
```
Comment (peterdmv) [inline HOWTO/INSTALL-WIN32.md:181]: export ERL_TOP=$HOME/src/otp_src_%OTP-REL%

**Pull request merged**